### PR TITLE
feat/new-release-news

### PR DESCRIPTION
**Pull request by danielaiozzo**

### CODE REVIEW

File: _posts/2022-12-20-Ecco-l-anteprima-del-nuovo-Designers-Italia [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+homeBottomLink: false
+homeHideCategories: true
+date_prepend: "Articolo"
+categories:
+  - cittadini
+  - pubblica amministrazione
+---
+
+<!-- MORE -->
+**Una Pubblica Amministrazione più vicina alle persone vicina, semplice e utile: [prossima.designers.italia.it](/)** non è solo **la prossima iterazione** del sito di Designers Italia, il progetto del Dipartimento per la trasformazione digitale e di Agenzia per l’Italia digitale (AGID), ma soprattutto un’ulteriore evoluzione dell’iniziativa che **crea e diffonde strumenti pratici** per progettare e realizzare servizi pubblici digitali di qualità e **cura una community** che si confronta sulle buone pratiche di settore.
```
Comment (Fupete) [inline _posts/2022-12-20-Ecco-l-anteprima-del-nuovo-Designers-Italia:19]: due volte vicina @danielaiozzo @gianvitofanelli

**Review by Fupete — CHANGES_REQUESTED** [review]

**Review by Fupete — APPROVED** [review]

**Comment by Fupete** [comment]
@danielaiozzo merge yourself when needed 🙇

File: _posts/2022-12-20-Ecco-l-anteprima-del-nuovo-Designers-Italia [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+homeBottomLink: false
+homeHideCategories: true
+date_prepend: "Articolo"
+categories:
+  - cittadini
+  - pubblica amministrazione
+---
+
+<!-- MORE -->
+**Una Pubblica Amministrazione più vicina alle persone, semplice e utile: [prossima.designers.italia.it](/)** non è solo **la prossima iterazione** del sito di Designers Italia, il progetto del Dipartimento per la trasformazione digitale e di Agenzia per l’Italia digitale (AGID), ma soprattutto un’ulteriore evoluzione dell’iniziativa che **crea e diffonde strumenti pratici** per progettare e realizzare servizi pubblici digitali di qualità e **cura una community** che si confronta sulle buone pratiche di settore.
```
Comment (Fupete) [inline _posts/2022-12-20-Ecco-l-anteprima-del-nuovo-Designers-Italia:19]: you need https:// at the beginning of the url... @danielaiozzo

File: _posts/2022-12-20-Ecco-l-anteprima-del-nuovo-Designers-Italia [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+Grazie ad analisi euristiche approfondite e ai dati di utilizzo di sito e risorse negli ultimi anni, abbiamo **riprogettato completamente l’architettura informativa**, rivisto i **modelli dei contenuti** e **l’esperienza di navigazione**. Lo abbiamo fatto pensando alle necessità di chi utilizza il sito e mettendoci nei loro panni: le persone della PA, i fornitori e la community di designer e developer impegnate nella trasformazione della PA e che, come ha testimoniato il successo degli ultimi eventi fisici e digitali, segue con attenzione e partecipazione questo progetto.
+
+Due le grandi novità. La prima: già visibile in anteprima all'indirizzo [prossima.designers.italia.it](/), e che sarà arricchita di contenuti nei prossimi mesi, è la sezione che contiene il **Design system del Paese**, la cassetta degli attrezzi per progettare e realizzare esperienze di servizi pubblici digitali di qualità.
```
Comment (Fupete) [inline _posts/2022-12-20-Ecco-l-anteprima-del-nuovo-Designers-Italia:23]: Here too.... https:/

**Review by Fupete — CHANGES_REQUESTED** [review]

File: _posts/2022-12-20-Ecco-l-anteprima-del-nuovo-Designers-Italia (hunk 2 above)
Comment (Fupete) [reply _posts/2022-12-20-Ecco-l-anteprima-del-nuovo-Designers-Italia:19]: ops you need the full url: https://prossima.designers.italia.it on the right between ()

Comment (Fupete) [reply _posts/2022-12-20-Ecco-l-anteprima-del-nuovo-Designers-Italia:19]: the text between [] is ok instead

Comment (danielaiozzo) [reply _posts/2022-12-20-Ecco-l-anteprima-del-nuovo-Designers-Italia]: done

File: _posts/2022-12-20-Ecco-l-anteprima-del-nuovo-Designers-Italia (hunk 3 above)
Comment (danielaiozzo) [reply _posts/2022-12-20-Ecco-l-anteprima-del-nuovo-Designers-Italia]: done

**Comment by bfabio** [comment]
@Fupete looks like the filename is missing the `.md` extension

**Review by Fupete — APPROVED** [review]

**Pull request merged**